src/maestro_mcp/server.py

```diff
@@ -15,9 +15,9 @@
 from starlette.responses import PlainTextResponse
 from pydantic import BaseModel, Field
 
-from ..chunking import ChunkingConfig
-from ..db.vector_db_base import VectorDatabase
-from ..db.vector_db_factory import create_vector_database
+from src.chunking import ChunkingConfig
+from src.db.vector_db_base import VectorDatabase
+from src.db.vector_db_factory import create_vector_database
```

Comment:
why are you removing the relative imports from this file? If we don't want to use relative imports then we should be exporting the imports in init file (iiuc)

Reply:
problematic import when adding tests. I'd be inclined to fix imports with __init.py__ fully in a subsequent PR, as it can lead to a number of changes. This change appears to work for normal usage & the tests we currently have in place